lib/features/support/env.rb

```diff
@@ -89,25 +89,13 @@ def encode_query_params hash
   URI.encode_www_form hash
 end
 
-def run_command(cmd, must_pass=true)
-  command_status = nil
-  command_out = Set.new
+def run_command(*cmd, must_pass: true)
   STDOUT.puts cmd if ENV['VERBOSE']
-  Open3.popen3(cmd) do |stdin, stdout, stderr, thread|
-    { :out => stdout, :err => stderr }.each do |key, stream|
-      Thread.new do
-        until (output = stream.gets).nil? do
-          command_out << output
-          STDOUT.puts output if ENV['VERBOSE']
-        end
-      end
-    end
+  stdout, stderr, status = Open3.capture3(*cmd)
+  STDOUT.puts stdout if ENV['VERBOSE']
+  STDOUT.puts stderr if ENV['VERBOSE']
 
-    thread.join
-    command_status = thread.value
-  end
-  assert_true(command_status.success?) if must_pass
-  command_out
+  assert_true(status.success?) if must_pass
```

Comment:
Not having stdout return is breaking some of the docker functionality atm, can we return `stdout.split("\n")`?

Reply:
Sure we can return that. I only removed the return as it didn't seem like any of the methods were using the return value. What functionality broke?

Reply:
The `find_default_docker_compose` method needs it to detect whether the compose file was valid, and the `test_service_running` method needs it to determine whether the instance is actually running.

Reply:
cool, thanks for the info. Does what I added address that?